numpy/core/src/umath/simd.inc.src

```diff
@@ -1367,6 +1377,9 @@ static NPY_GCC_OPT_3 NPY_GCC_TARGET_@ISA@ void
         op += num_lanes;
         num_remaining_elements -= num_lanes;
     }
+
+    if (@mask_to_int@(overflow_mask))
+        _mm_setcsr(_mm_getcsr() | (0x1 << 3));
```

Comment:
we have a function for that: `npy_set_floatstatus_overflow`

Reply:
thanks :)